tweetbot.go

```diff
@@ -36,56 +38,93 @@ func check(e error) {
 func WriteToFile(filename string, data string) error {
 	file, err := os.Create(filename)
 	if err != nil {
-		return err
+		return fmt.Errorf("could not open file: %s", err)
 	}
-	defer file.Close()
+	defer closeFile(file)
 
 	_, err = io.WriteString(file, data)
-	check(err)
-	return file.Sync()
```

Comment:
the file.Sync is not necessary, just by Closing the content will be written to the file system